OOInteraction/src/OOInteractionPlugin.cpp

```diff
@@ -162,6 +163,8 @@ bool OOInteractionPlugin::initialize(Core::EnvisionManager&)
 	OOInteraction::DoxygenWholeTreeVisitor::init();
 	OOInteraction::DoxygenCommentsOnlyVisitor::init();
 
+	Interaction::GenericHandler::instance()->addCommand(new AddNodeCommand());
```

Comment:
I doubt we want to include this for all node types. Some things don't make sense, e.g. a single literal.

For now why don't we register this for all declarations only?